src/components/TextField/TextField.scss

```diff
@@ -170,6 +170,22 @@ $stacking-order: (
   pointer-events: none;
 }
 
+.CharacterCount {
+  @include text-emphasis-subdued;
+  z-index: z-index(contents, $stacking-order);
```

Comment:
Is the `z-index` necessary to specify here?

Reply:
This is needed because of https://github.com/Shopify/polaris-react/blob/master/src/components/TextField/TextField.scss#L140 which sets a `z-index`